ejb-txn-remote-call/README-source.adoc

```diff
@@ -727,51 +727,32 @@ wildfly-operator-5d4b7cc868-zfxcv   1/1     Running     1          22h
 
-[NOTE]
+[WARNING]
 ====
-* You have access to an OpenShift Container Platform cluster using an account with cluster-admin permissions.
-* You have installed the oc tool in your local system.
+For simplicity, this quickstart installs the EAP operator globally in the `openshift-operators` namespace, making it available across all namespaces in the cluster. The `openshift-operators` namespace already contains a `global-operators` OperatorGroup, so you must NOT create an additional OperatorGroup or the operator will fail to start. In production environments, consider namespace-scoped installation (see note below) to limit scope and improve security.
 ====
 
-. View the list of operators available to the cluster from the OperatorHub:
-+
-[source,sh,options="nowrap",subs="+quotes,attributes+"]
-----
-oc get packagemanifests -n openshift-marketplace | grep eap
-----
-. Create a `Subscription` object YAML file (for example, `eap-operator-sub.yaml`) to subscribe a namespace to your EAP operator. The following is an example `Subscription` object YAML file:
-+
-[source,sh,options="nowrap",subs="+quotes,attributes+"]
-----
-apiVersion: operators.coreos.com/v1alpha1
-kind: Subscription
-metadata:
-  name: eap
-  namespace: openshift-operators
-spec:
-  channel: stable
-  installPlanApproval: Automatic
-  name: eap 
-  source:  redhat-operators 
-  sourceNamespace: openshift-marketplace
-----
-. Create the `Subscription` object from the YAML file:
-+
+. Log in into the Openshift console as `kubeadmin` user (or any cluster administrator).
+. Install the EAP operator globally.
+
+In your terminal, run the following command to subscribe the EAP operator.
 [source,sh,options="nowrap",subs="+quotes,attributes+"]
 ----
-oc apply -f eap-operator-sub.yaml
+$ oc apply -f charts/eap-operator-sub.yaml --wait --timeout=10m0s
 ----
-+
-The EAP operator is successfully installed. At this point, the OLM is aware of the EAP operator.
-A ClusterServiceVersion (CSV) for the operator appears in the target namespace, and APIs provided by the EAP operator is available for creation.
-To verify that the {productName} Operator is running, execute the following command:
-+
+
+To verify the operator is ready to use, you can run the following command. You should see the following output.
+The `PHASE` needs to be `Succeeded`
 [source,sh,options="nowrap",subs="+quotes,attributes+"]
 ----
-oc get csv -n openshift-operators
+$ oc get ClusterServiceVersion -n openshift-operators
 ----
+
+NOTE: For namespace-scoped installation: Edit `charts/eap-operator-group.yaml` to replace `<your-namespace>` with your project namespace, and edit `charts/eap-operator-sub.yaml` to replace `openshift-operators` with your project namespace. Then apply both files: `oc apply -f charts/eap-operator-group.yaml && oc apply -f charts/eap-operator-sub.yaml`
```

Comment:
I think this note should be at line 744 before `To verify the operator is ready to use, you can run the following command. You should see the following output.`

Reply:
is it different in eap 8.1 branch?